projects/step-frontend/src/lib/modules/timeseries/util/filter-utils.ts

```diff
@@ -7,47 +7,56 @@ export class FilterUtils {
     );
   }
 
-  static objectToOQL(object: { [key: string]: string }, attributesPrefix?: string) {
+  static objectToOQL(
+    object: { [key: string]: string },
+    attributesPrefix?: string,
+    attributeProcessFn?: (x: string) => string
```

Comment:
try to avoid having names like ```x```, instead try to use the actual name of the thing (looks like the name should be ```attribute```)